sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/TestConfigurations.java

```diff
@@ -32,26 +32,28 @@
     private static Logger logger = LoggerFactory.getLogger(TestConfigurations.class);
     private static Properties properties = loadProperties();
 
+    public final static String COSMOS_EMULATOR_KEY = "C2y6yDjf5/R+ob0N8A7Cgv30VRDJIWEHLM+4QDU5DE2nQ9nDuVTqobD4b8mGGyPMbIZnqyMsEcaGQy67XIw/Jw==";
+    public final static String COSMOS_EMULATOR_HOST = "https://localhost:8081/";
```

Comment:
why are we defining new public property for something which is already available?

I thought the plan was to reuse `TestConfiguration.HOST`

on the CI the value for TestConfiguration.HOST will be populated by the CI.
in local debugging that can be populated from a properties file from user home.

This breaks that.